onnx/onnx.in.proto

```diff
@@ -446,8 +446,6 @@ message TypeProto {
     optional string domain = 1;
     // The name is optional but significant when provided.
     optional string name = 2;
-    // parameters that help defining the type
-    repeated TypeProto parameters = 3;
```

Comment:
Technically we need to make sure this field number won't be accidentally reused in the future (otherwise it will break backward compatibility **silently**), so what we used to do is comment out this line (instead of removing) and add a comment saying do not use it.